src/source/PeerConnection/Rtp.c

```diff
@@ -31,6 +31,28 @@ STATUS createKvsRtpTransceiver(RTC_RTP_TRANSCEIVER_DIRECTION direction, PKvsPeer
     pKvsRtpTransceiver->transceiver.receiver.track.kind = pRtcMediaStreamTrack->kind;
     pKvsRtpTransceiver->transceiver.direction = direction;
 
+    if (rollingBufferDurationSec < MIN_ROLLING_BUFFER_DURATION_IN_SECONDS) {
+        DLOGW("Rolling buffer duration set to less than 100 ms. Setting to default %d sec", DEFAULT_ROLLING_BUFFER_DURATION_IN_SECONDS);
+        pKvsRtpTransceiver->rollingBufferDurationSec = DEFAULT_ROLLING_BUFFER_DURATION_IN_SECONDS;
+    } else {
+        pKvsRtpTransceiver->rollingBufferDurationSec = rollingBufferDurationSec;
+    }
+    if (rollingBufferBitratebps < MIN_EXPECTED_BIT_RATE) {
+        if (pRtcMediaStreamTrack->kind == MEDIA_STREAM_TRACK_KIND_VIDEO) {
+            DLOGW("Rolling buffer duration set to less than 100 KBps for video. Setting to default %d Bps", DEFAULT_EXPECTED_VIDEO_BIT_RATE);
+            pKvsRtpTransceiver->rollingBufferBitratebps = DEFAULT_EXPECTED_VIDEO_BIT_RATE;
+        } else if (pRtcMediaStreamTrack->kind == MEDIA_STREAM_TRACK_KIND_AUDIO) {
+            DLOGW("Rolling buffer duration set to less than 100 KBps for audio. Setting to default %d Bps", DEFAULT_EXPECTED_AUDIO_BIT_RATE);
+            pKvsRtpTransceiver->rollingBufferBitratebps = DEFAULT_EXPECTED_AUDIO_BIT_RATE;
+        } else {
+            DLOGW("Rolling buffer duration set to less than 100 KBps for unknown codec. Setting to default %d Bps", DEFAULT_EXPECTED_AUDIO_BIT_RATE);
```

Comment:
Careful with caps. Capital B usually represents bytes.

Also minor: Ki as prefix since increments of 1024 instead of 1000